tests/h/formatters/annotation_flag_test.py

```diff
@@ -1,12 +1,8 @@
-from collections import namedtuple
-
 import pytest
 
 from h.formatters.annotation_flag import AnnotationFlagFormatter
 from h.services.flag import FlagService
 
-FakeAnnotationContext = namedtuple("FakeAnnotationContext", ["annotation"])
```

Comment:
All of the formatter tests were using a named tuple instead of the real `AnnotationContext` or an autospec for some reason. We don't need that and it also means they drifted from the actual interface of the context object.